src/common/libjob/specutil.c

```diff
@@ -38,10 +39,13 @@ json_t *specutil_argv_create (int argc, char **argv)
     return NULL;
 }
 
-int specutil_env_set (json_t *o, const char *name, const char *value)
+int specutil_env_set (json_t *o, const char *name, const char *value, int overwrite)
```

Comment:
Exceeds 80 cols so best to break up to one param per line

[RFC7](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_7.html) recommendation 5